src/cache/clear_site_data/directive.rs

```diff
@@ -0,0 +1,59 @@
+use std::fmt::Display;
+use std::str::FromStr;
+
+/// An HTTP `Clear-Site-Data` directive.
+///
+/// [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data#Directives)
+#[derive(Debug, Eq, PartialEq, Hash, Clone, Copy)]
+pub enum ClearDirective {
+    /// Indicates that the server wishes to remove locally cached data (i.e. the
+    /// browser cache, see HTTP caching) for the origin of the response URL.
+    /// Depending on the browser, this might also clear out things like
+    /// pre-rendered pages, script caches, WebGL shader caches, or address bar
+    /// suggestions.
+    Cache,
+    /// Indicates that the server wishes to remove all cookies for the origin of
+    /// the response URL. HTTP authentication credentials are also cleared out.
+    /// This affects the entire registered domain, including subdomains. So
+    /// https://example.com as well as https://stage.example.com, will have
+    /// cookies cleared.
+    Cookies,
+    /// Indicates that the server wishes to remove all DOM storage for the origin
+    /// of the response URL.
+    Storage,
+    /// Indicates that the server wishes to reload all browsing contexts for the
+    /// origin of the response (Location.reload).
+    ExecutionContexts,
+}
+
+impl ClearDirective {
+    /// Get the formatted string.
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            ClearDirective::Cache => r#""cache""#,
+            ClearDirective::Cookies => r#""cookies""#,
+            ClearDirective::Storage => r#""storage""#,
+            ClearDirective::ExecutionContexts => r#""executionContexts""#,
```

Comment:
```suggestion
            ClearDirective::Cache => "cache",
            ClearDirective::Cookies => "cookies",
            ClearDirective::Storage => "storage",
            ClearDirective::ExecutionContexts => "executionContexts",
```

Why use rust string syntax when unnecessary?

Reply:
The quotes here are intentional! Each value **must** include quotes, so it's strings like `"cookies", "storage"`. Hence the usage of raw strings. There's a test for this at the bottom of the file.

Reply:
Ohhh! I see. Sorry!